src/components/breadcrumbs/_breadcrumb_content.tsx

```diff
@@ -70,10 +69,14 @@ export const EuiBreadcrumbContent: FunctionComponent<
 
   const isBreadcrumbWithPopover = !!popoverContent;
   const isInteractiveBreadcrumb = href || onClick;
-  const linkColor = color || (highlightLastBreadcrumb ? 'text' : 'subdued');
-  const plainTextColor = highlightLastBreadcrumb ? 'default' : 'subdued'; // Does not inherit `color` prop
+  const linkColor = color || 'subdued';
   const ariaCurrent = highlightLastBreadcrumb ? ('page' as const) : undefined;
 
+  const interactionStyles = [
+    (isInteractiveBreadcrumb || isBreadcrumbWithPopover) &&
+      styles.isInteractive,
+  ];
```

Comment:
[very optional, feel free to skip if you prefer the spread consistency]

since we're only adding 1 interactive style total, we could skip the array format:
```suggestion
  const interactionStyles =
    (isInteractiveBreadcrumb || isBreadcrumbWithPopover) &&
    styles.isInteractive;
```

and down below we'd skip the spread, e.g. `css={[...cssStyles, interactiveStyles]}`